blog/release-notes.md

```
@@ -0,0 +1,67 @@
+---
+slug: /release-notes
+tags: [release]
+pinned: true
+---
+
+# Release Notes
+
+### Version 1.3.0 - August 28, 2025
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)